fastwhisper_server/README.md

```diff
@@ -0,0 +1,116 @@
+# FastWhisper Integration for Lotti
+
+This integration provides speech-to-text transcription capabilities for the Lotti application using FastWhisper, a high-performance speech recognition system.
+
+## Overview
+
+FastWhisper is integrated into Lotti to provide efficient and accurate audio transcription. The system consists of two main components:
+
+1. A FastAPI server that handles the transcription requests
+2. A Flutter client service that communicates with the server
+
+## Features
+
+- Real-time audio transcription
+- Multiple model support (small, base, large-v3)
+- Automatic language detection
+- Progress tracking during transcription
+- Segment-level transcription with timestamps
+- Fallback to WhisperKit when FastWhisper is unavailable
```

Comment:
Please remove this line, that's not something for this service here, that's only specific to the call site. E.g. when we remove WhisperKit altogether from Lotti when this service is stable, we should not have to come back here and update the README